### PR TITLE
Test enhancement

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log

- set the correct Travis CI settings. See the latest Travis CI [build](https://travis-ci.org/peter279k/Transliterator/builds/339572153).
- enhance the tests.

### CODE REVIEW

File: tests/Transliterator/invalid_map.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * (c) Саша Стаменковић <umpirsky@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Transliterator;
+
+/**
+ * Belarusian mapping (ALA-LC system).
```
Comment (umpirsky) [inline tests/Transliterator/invalid_map.php]: Remove wrong docblock. :)

File: src/Transliterator/data/be/default.php [hunk 2]
```diff
@@ -9,6 +9,8 @@
  * file that was distributed with this source code.
  */
 
+namespace Transliterator;
```
Comment (umpirsky) [inline src/Transliterator/data/be/default.php]: Why?

Comment (peter279k) [reply src/Transliterator/data/be/default.php]: I think add this to identify the source code is under the ```Transliterator``` namespace.

Comment (umpirsky) [reply src/Transliterator/data/be/default.php]: There is no need, is there?

Comment (peter279k) [reply src/Transliterator/data/be/default.php]: Oh, you're right... The namespace is no need there.
The map files in ```data``` folder are require directly.
So I have to remove these namespace in all map files.

**Comment by peter279k** [comment]
Hi @umpirsky, I have already remove the wrong namespace and docblock.
If having any changes, please feel free to send the request chnages :).

**Comment by umpirsky** [comment]
Neat, thanks @peter279k!

**Pull request merged**